### ISSUE TITLE
quickscreenshot + clipboard

### ISSUE THREAD

**Issue by Dibal** [xournalpp/xournalpp#3953]
The plugin "quickscreenshot" is a nice way to save parts of the xournal screen.

Question: is it possible to extend the Lua script by a command that copies the image to the clipboard (as well)?

**Comment by Technius** [comment]
Yes, you can modify the QuickScreenshot plugin so that it calls a command to copy to clipboard. For example, if you're using a Linux distribution and you have the [`maim`](https://github.com/naelstrof/maim) program installed, the image is copied to clipboard:
https://github.com/xournalpp/xournalpp/blob/6f151c7704a53efde1714d9a0d98eca7d597c783/plugins/QuickScreenshot/main.lua#L82-L84

**Comment by Dibal** [comment]
It works theoretically -- but the screenshot is spoiled by vertical an horizontal lines:
when you select area for screenshot the border lines of selection are not removed but added to the image.
